<!DOCTYPE html><html lang="zh-CN" data-theme="light"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0,viewport-fit=cover"><title>基因组-代谢组mGWAS联合分析综述 | 丸子Blog</title><meta name="author" content="wanzi"><meta name="copyright" content="wanzi"><meta name="format-detection" content="telephone=no"><meta name="theme-color" content="#ffffff"><meta name="description" content="基因组-代谢组mGWAS联合分析综述">
<meta property="og:type" content="article">
<meta property="og:title" content="基因组-代谢组mGWAS联合分析综述">
<meta property="og:url" content="http://example.com/2024/12/24/[%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB]%E5%9F%BA%E5%9B%A0%E7%BB%84-%E4%BB%A3%E8%B0%A2%E7%BB%84mGWAS%E7%BB%BC%E8%BF%B0/index.html">
<meta property="og:site_name" content="丸子Blog">
<meta property="og:description" content="基因组-代谢组mGWAS联合分析综述">
<meta property="og:locale" content="zh_CN">
<meta property="og:image" content="http://example.com/img/cover6.jpg">
<meta property="article:published_time" content="2024-12-24T11:55:32.000Z">
<meta property="article:modified_time" content="2024-12-27T10:14:15.990Z">
<meta property="article:author" content="wanzi">
<meta property="article:tag" content="多组学">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="http://example.com/img/cover6.jpg"><link rel="shortcut icon" href="/./img/favicon1.png"><link rel="canonical" href="http://example.com/2024/12/24/[%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB]%E5%9F%BA%E5%9B%A0%E7%BB%84-%E4%BB%A3%E8%B0%A2%E7%BB%84mGWAS%E7%BB%BC%E8%BF%B0/index.html"><link rel="preconnect" href="//cdn.jsdelivr.net"/><link rel="preconnect" href="//busuanzi.ibruce.info"/><link rel="stylesheet" href="/css/index.css?v=4.13.0"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.5.1/css/all.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fancyapps/ui@5.0.33/dist/fancybox/fancybox.min.css" media="print" onload="this.media='all'"><script>const GLOBAL_CONFIG = {
  root: '/',
  algolia: undefined,
  localSearch: {"path":"/search.xml","preload":false,"top_n_per_article":1,"unescape":false,"languages":{"hits_empty":"找不到您查询的内容：${query}","hits_stats":"共找到 ${hits} 篇文章"}},
  translate: undefined,
  noticeOutdate: undefined,
  highlight: {"plugin":"highlight.js","highlightCopy":true,"highlightLang":true,"highlightHeightLimit":false},
  copy: {
    success: '复制成功',
    error: '复制错误',
    noSupport: '浏览器不支持'
  },
  relativeDate: {
    homepage: true,
    post: true
  },
  runtime: '天',
  dateSuffix: {
    just: '刚刚',
    min: '分钟前',
    hour: '小时前',
    day: '天前',
    month: '个月前'
  },
  copyright: undefined,
  lightbox: 'fancybox',
  Snackbar: undefined,
  infinitegrid: {
    js: 'https://cdn.jsdelivr.net/npm/@egjs/infinitegrid@4.11.1/dist/infinitegrid.min.js',
    buttonText: '加载更多'
  },
  isPhotoFigcaption: false,
  islazyload: false,
  isAnchor: false,
  percent: {
    toc: true,
    rightside: false,
  },
  autoDarkmode: false
}</script><script id="config-diff">var GLOBAL_CONFIG_SITE = {
  title: '基因组-代谢组mGWAS联合分析综述',
  isPost: true,
  isHome: false,
  isHighlightShrink: false,
  isToc: true,
  postUpdate: '2024-12-27 18:14:15'
}</script><script>(win=>{
      win.saveToLocal = {
        set: (key, value, ttl) => {
          if (ttl === 0) return
          const now = Date.now()
          const expiry = now + ttl * 86400000
          const item = {
            value,
            expiry
          }
          localStorage.setItem(key, JSON.stringify(item))
        },
      
        get: key => {
          const itemStr = localStorage.getItem(key)
      
          if (!itemStr) {
            return undefined
          }
          const item = JSON.parse(itemStr)
          const now = Date.now()
      
          if (now > item.expiry) {
            localStorage.removeItem(key)
            return undefined
          }
          return item.value
        }
      }
    
      win.getScript = (url, attr = {}) => new Promise((resolve, reject) => {
        const script = document.createElement('script')
        script.src = url
        script.async = true
        script.onerror = reject
        script.onload = script.onreadystatechange = function() {
          const loadState = this.readyState
          if (loadState && loadState !== 'loaded' && loadState !== 'complete') return
          script.onload = script.onreadystatechange = null
          resolve()
        }

        Object.keys(attr).forEach(key => {
          script.setAttribute(key, attr[key])
        })

        document.head.appendChild(script)
      })
    
      win.getCSS = (url, id = false) => new Promise((resolve, reject) => {
        const link = document.createElement('link')
        link.rel = 'stylesheet'
        link.href = url
        if (id) link.id = id
        link.onerror = reject
        link.onload = link.onreadystatechange = function() {
          const loadState = this.readyState
          if (loadState && loadState !== 'loaded' && loadState !== 'complete') return
          link.onload = link.onreadystatechange = null
          resolve()
        }
        document.head.appendChild(link)
      })
    
      win.activateDarkMode = () => {
        document.documentElement.setAttribute('data-theme', 'dark')
        if (document.querySelector('meta[name="theme-color"]') !== null) {
          document.querySelector('meta[name="theme-color"]').setAttribute('content', '#0d0d0d')
        }
      }
      win.activateLightMode = () => {
        document.documentElement.setAttribute('data-theme', 'light')
        if (document.querySelector('meta[name="theme-color"]') !== null) {
          document.querySelector('meta[name="theme-color"]').setAttribute('content', '#ffffff')
        }
      }
      const t = saveToLocal.get('theme')
    
        if (t === 'dark') activateDarkMode()
        else if (t === 'light') activateLightMode()
      
      const asideStatus = saveToLocal.get('aside-status')
      if (asideStatus !== undefined) {
        if (asideStatus === 'hide') {
          document.documentElement.classList.add('hide-aside')
        } else {
          document.documentElement.classList.remove('hide-aside')
        }
      }
    
      const detectApple = () => {
        if(/iPad|iPhone|iPod|Macintosh/.test(navigator.userAgent)){
          document.documentElement.classList.add('apple')
        }
      }
      detectApple()
    })(window)</script><meta name="generator" content="Hexo 7.3.0"><link rel="alternate" href="/atom.xml" title="丸子Blog" type="application/atom+xml">
</head><body><div id="sidebar"><div id="menu-mask"></div><div id="sidebar-menus"><div class="avatar-img is-center"><img src="/./img/avatar.jpg" onerror="onerror=null;src='/img/friend_404.gif'" alt="avatar"/></div><div class="sidebar-site-data site-data is-center"><a href="/archives/"><div class="headline">文章</div><div class="length-num">41</div></a><a href="/tags/"><div class="headline">标签</div><div class="length-num">35</div></a><a href="/categories/"><div class="headline">分类</div><div class="length-num">10</div></a></div><hr class="custom-hr"/><div class="menus_items"><div class="menus_item"><a class="site-page" href="/"><i class="fa-fw fas fa-home"></i><span> 主页</span></a></div><div class="menus_item"><a class="site-page" href="/archives/"><i class="fa-fw fas fa-archive"></i><span> 归档</span></a></div><div class="menus_item"><a class="site-page" href="/tags/"><i class="fa-fw fas fa-tags"></i><span> 标签</span></a></div><div class="menus_item"><a class="site-page" href="/categories/"><i class="fa-fw fas fa-folder-open"></i><span> 分类</span></a></div><div class="menus_item"><a class="site-page group" href="javascript:void(0);"><i class="fa-fw fas fa-list"></i><span> 链接</span><i class="fas fa-chevron-down"></i></a><ul class="menus_item_child"><li><a class="site-page child" href="/link/"><i class="fa-fw fas fa-link"></i><span> 友情链接</span></a></li><li><a class="site-page child" href="/about/"><i class="fa-fw fas fa-heart"></i><span> 关于</span></a></li></ul></div></div></div></div><div class="post" id="body-wrap"><header class="post-bg" id="page-header" style="background-image: url('/img/cover6.jpg')"><nav id="nav"><span id="blog-info"><a href="/" title="丸子Blog"><span class="site-name">丸子Blog</span></a></span><div id="menus"><div id="search-button"><a class="site-page social-icon search" href="javascript:void(0);"><i class="fas fa-search fa-fw"></i><span> 搜索</span></a></div><div class="menus_items"><div class="menus_item"><a class="site-page" href="/"><i class="fa-fw fas fa-home"></i><span> 主页</span></a></div><div class="menus_item"><a class="site-page" href="/archives/"><i class="fa-fw fas fa-archive"></i><span> 归档</span></a></div><div class="menus_item"><a class="site-page" href="/tags/"><i class="fa-fw fas fa-tags"></i><span> 标签</span></a></div><div class="menus_item"><a class="site-page" href="/categories/"><i class="fa-fw fas fa-folder-open"></i><span> 分类</span></a></div><div class="menus_item"><a class="site-page group" href="javascript:void(0);"><i class="fa-fw fas fa-list"></i><span> 链接</span><i class="fas fa-chevron-down"></i></a><ul class="menus_item_child"><li><a class="site-page child" href="/link/"><i class="fa-fw fas fa-link"></i><span> 友情链接</span></a></li><li><a class="site-page child" href="/about/"><i class="fa-fw fas fa-heart"></i><span> 关于</span></a></li></ul></div></div><div id="toggle-menu"><a class="site-page" href="javascript:void(0);"><i class="fas fa-bars fa-fw"></i></a></div></div></nav><div id="post-info"><h1 class="post-title">基因组-代谢组mGWAS联合分析综述</h1><div id="post-meta"><div class="meta-firstline"><span class="post-meta-date"><i class="far fa-calendar-alt fa-fw post-meta-icon"></i><span class="post-meta-label">发表于</span><time class="post-meta-date-created" datetime="2024-12-24T11:55:32.000Z" title="发表于 2024-12-24 19:55:32">2024-12-24</time><span class="post-meta-separator">|</span><i class="fas fa-history fa-fw post-meta-icon"></i><span class="post-meta-label">更新于</span><time class="post-meta-date-updated" datetime="2024-12-27T10:14:15.990Z" title="更新于 2024-12-27 18:14:15">2024-12-27</time></span><span class="post-meta-categories"><span class="post-meta-separator">|</span><i class="fas fa-inbox fa-fw post-meta-icon"></i><a class="post-meta-categories" href="/categories/%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB/">论文阅读</a></span></div><div class="meta-secondline"><span class="post-meta-separator">|</span><span class="post-meta-wordcount"><i class="far fa-file-word fa-fw post-meta-icon"></i><span class="post-meta-label">字数总计:</span><span class="word-count">10.2k</span><span class="post-meta-separator">|</span><i class="far fa-clock fa-fw post-meta-icon"></i><span class="post-meta-label">阅读时长:</span><span>53分钟</span></span><span class="post-meta-separator">|</span><span class="post-meta-pv-cv" id="" data-flag-title="基因组-代谢组mGWAS联合分析综述"><i class="far fa-eye fa-fw post-meta-icon"></i><span class="post-meta-label">阅读量:</span><span id="busuanzi_value_page_pv"><i class="fa-solid fa-spinner fa-spin"></i></span></span></div></div></div></header><main class="layout" id="content-inner"><div id="post"><article class="post-content" id="article-container"><p>[toc]</p>
<h1>代谢组学综述</h1>
<p><a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/10.1111/tpj.14097">Metabolic GWAS-based dissection of genetic bases underlying the diversity of plant metabolism</a></p>
<p><a href="">Metabolite-based genome-wide association studies in plants</a></p>
<p>focus水果：<a target="_blank" rel="noopener" href="https://www.sciencedirect.com/science/article/pii/S0168945223001656?via%3Dihub">Limitations and advantages of using metabolite-based genome-wide association studies: Focus on fruit quality traits</a></p>
<p>focus植物育种：<a href="">Recent applications of metabolomics in plant breeding</a></p>
<p>focus果树:<a target="_blank" rel="noopener" href="https://webofscience.clarivate.cn/wos/alldb/full-record/WOS:000796372500001">Research Progress and Trends in Metabolomics of Fruit Trees</a></p>
<p><a target="_blank" rel="noopener" href="https://webofscience.clarivate.cn/wos/alldb/full-record/WOS:000454428700012">Exploring the Diversity of Plant Metabolism</a></p>
<p><a target="_blank" rel="noopener" href="https://webofscience.clarivate.cn/wos/alldb/full-record/WOS:000595005100001">Metabolomics: a systems biology approach for enhancing heat stress tolerance in plants</a></p>
<p>Metabolic GWAS-based dissection of genetic bases underlying the diversity of plant metabolism</p>
<p>NC:<a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41467-022-33238-4">GWAS, MWAS and mGWAS provide insights into precision agriculture based on genotype-dependent microbial effects in foxtail millet</a></p>
<h1>Metabolic GWAS-based dissection of genetic bases underlying the diversity of plant metabolism</h1>
<p>2018年 《基于代谢 GWAS 的植物代谢多样性遗传基础剖析》</p>
<p>基于<strong>质谱</strong>的分析系统和<strong>基因组测序技术</strong>的发展。GWAS的概念扩展到了代谢组的研究，产生了<strong>mGWAS</strong>，是全球识别植物代谢多样性遗传决定因素的最强大工具之一。</p>
<p>最近，mGWAS已经在<strong>各种物种</strong>中进行了持续改进，提供了对代谢多样性遗传基础的更深刻见解。</p>
<ul>
<li>我们首先总结了GWAS及其基于统计方法和群体的发展。</li>
<li>由于靶向性状的变异对GWAS至关重要，我们回顾了代谢多样性及其在群体和个体层面的兴起。</li>
<li>随后，讨论了mGWAS在植物中的应用及其相应的成就。</li>
<li>我们还讨论了基于mGWAS的多维分析的当前知识以及对代谢多样性的新见解。</li>
</ul>
<h2 id="Introduction">Introduction</h2>
<p>每个植物个体的代谢组受到其发育阶段、组织和环境刺激的显著影响</p>
<blockquote>
<p>植物由于其定居性，能够产生<strong>数量庞大的代谢物</strong>，估计有10万到100万种，但<strong>每个物种仅生产其中的一部分</strong>。大多数代谢物在不同物种中表现出特异性的积累。此外，研究发现，即使<strong>在同一物种内</strong>，<strong>代谢物的种类和数量的变异</strong>也比之前预期的要大得多。这些代谢物的生成受到植物的发育阶段、不同组织以及环境条件的显著影响。</p>
</blockquote>
<ul>
<li>
<p>要点</p>
<ul>
<li>
<p>植物代谢组的多样性源于基因组进化、遗传多样性、发育阶段和环境刺激。</p>
</li>
<li>
<p>mGWAS是识别影响代谢多样性的常见遗传变异的强大工具。</p>
</li>
<li>
<p>通过精心设计的方法和/或群体来发展GWAS，将为解码代谢多样性的遗传基础（包括罕见变异）提供新的机会。</p>
</li>
<li>
<p>基于mGWAS的多维数据分析对于增强我们对代谢多样性及其相应遗传基础的理解至关重要。</p>
</li>
</ul>
</li>
<li>
<p><strong>未解问题</strong></p>
<ul>
<li>我们如何高效地识别结构相似但功能不同的代谢物的生物功能多样性？</li>
<li>不同修饰对代谢物的生化和生物功能有何影响，是否存在普遍规律？</li>
<li>我们如何全面解析代谢途径并以转录后、翻译后和/或表观遗传的方式表征基因功能？</li>
<li>基于不同条件下各个个体代谢物比例的mGWAS对于解析代谢响应环境刺激的机制是否具有强大效力？</li>
</ul>
</li>
</ul>
<h2 id="The-Development-of-GWAS">The Development of GWAS</h2>
<p>2005 年首次成功应用 GWAS，然后被广泛应用于解析复杂表型的遗传基础，涵盖了人类和动物的疾病，以及植物的生理和农业性状。</p>
<ul>
<li>刚开始存在的挑战
<ul>
<li>群体结构和亲缘关系导致的虚假关联问题。</li>
<li>混合线性模型（MLM）作为主要方法，有效纠正偏差。</li>
<li>开发多位点混合模型以应对复杂性状。</li>
</ul>
</li>
<li>局限性和创新
<ul>
<li>SNP 基于的 GWAS 面临多重测试惩罚和忽视 SNP 互作的问题</li>
<li>开发了单倍型基于 GWAS、基因基于 GWAS 和基于 A−D 检验的 GWAS，以提高识别能力和统计功效。</li>
</ul>
</li>
</ul>
<h2 id="The-Rise-of-Metabolic-Diversity">The Rise of Metabolic Diversity</h2>
<p>展示了不同物种的GWAS研究数量</p>
<p><img src="https://pic.imgdb.cn/item/676a5d53d0e0a243d4e92662.png" alt=""></p>
<p>随着next-generation sequencing和high-throughput phenotype screening筛选 technologies的进步，GWAS全面研究了各种性状，从individual morphological形态scale尺度  (plant height, yield, heading date, etc.)到interaction with various environmental stimuli(abiotic or biotic stresses)</p>
<p>但是呢，GWAS研究出来的QTLs有限，因为相比于代谢物之间的variations来说，不同品种间性状的variation非常的狭隘，代谢物的就非常丰富、而且大部分都有定量</p>
<ul>
<li>植物的代谢多样性是基因组进化和遗传多样性以及环境刺激的结果。
<ul>
<li>Development- and environment-dependent diversity of metabolites may be associated with their biochemical and biological functions resulting from spatial−temporal and/or environment-effected expression of corresponding genetic determinants.</li>
<li><strong>植物中各种化学物质的多样性会随着植物的生长阶段和所处环境的变化而变化。这些变化影响了这些化学物质在植物中的具体作用，因为不同的基因会在不同的时间、地点或环境条件下被激活或关闭。</strong></li>
</ul>
</li>
</ul>
<table>
<thead>
<tr>
<th><strong>类别</strong></th>
<th><strong>内容</strong></th>
<th><strong>例子/研究引用</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>代谢多样性的来源</strong></td>
<td>植物代谢多样性源于基因组进化、遗传多样性以及环境刺激。</td>
<td>-</td>
</tr>
<tr>
<td><strong>特定代谢物类：Cucurbitacins</strong>葫芦素</td>
<td>- 结构特征：四环cucurbitane核骨架的高度多样化三萜类化合物。 - 分类：已发现12类，包括Cucurbitacins A–T。 - 积累特性：各物种主要苦味化合物不同，如Cucurbitacin C仅在黄瓜中发现。 - 遗传基础：CYP基因的保守合成位点及物种特异性CYP缺失。</td>
<td>Chen et al., 2005; Panosyan et al., 1985; Da Costa and Jones, 1971; Shang et al., 2014; Zhou et al., 2016</td>
</tr>
<tr>
<td>**特定代谢物类：Strigolactones (SLs)**独脚金内酯</td>
<td>- 功能：调控植物整体架构的植物激素。 - 合成途径：all-trans-β-carotene异构化为9-cis-β-carotene，进一步转化为carlactone (CL)。 - 关键酶：细胞色素P450 AtMAX1对活性SL的合成至关重要。 - 遗传变异：两个AtMAX1同源基因的重排导致SL含量变异。</td>
<td>Gomez-Roldan et al., 2008; Umehara et al., 2008; Alder et al., 2012; Waters et al., 2012; Booker et al., 2005; Cardoso et al., 2014</td>
</tr>
<tr>
<td><strong>基因组进化机制</strong></td>
<td>- 机制：转座子插入、基因重复与发散、基因丧失等。 - 影响：促进代谢路径的分歧进化，导致代谢物多样性。</td>
<td>Kobayashi et al., 2004; Xu et al., 2017; Zhang, 2003; Palmieri et al., 2011</td>
</tr>
<tr>
<td><strong>作物改良的影响</strong></td>
<td>- 人类选择压力影响代谢进化。 - 例子：番茄育种中选择低苦味水果，减少苦味化合物含量。</td>
<td>Zhou et al., 2016; Zhu et al., 2018</td>
</tr>
<tr>
<td><strong>个体代谢多样性的决定因素</strong></td>
<td>- 决定方式：发育依赖和环境刺激依赖。 - 代谢物多样性通过糖基化和酰基化等修饰反应产生定性多样性。 - 例子：Luteolin 6-C-glucoside在水稻叶片中呈发育梯度累积。</td>
<td>Koes et al., 1994; Stobiecki, 2000; Lepiniec et al., 2006; Dong et al., 2015</td>
</tr>
<tr>
<td><strong>环境刺激对代谢的影响</strong></td>
<td>- 影响因素：干旱、营养缺乏、光照、温度等。 - 代谢反应：不同环境刺激诱导特定代谢物的积累。 - 例子：冷适应下20种类黄酮flavonoids的累积增加；病原体感染诱导epigallocatechin等flavonoids的积累。</td>
<td>Sperdouli and Moustakas, 2012; Jiang and Fu, 2007; Zoratti et al., 2014; Jaakola and Hohtola, 2010; Schulz et al., 2015; Koskimäki et al., 2009</td>
</tr>
</tbody>
</table>
<ul>
<li>Cucurbitacins</li>
</ul>
<p><img src="https://pic.imgdb.cn/item/676a63c7d0e0a243d4e9342b.png" alt=""></p>
<h2 id="Deciphering-the-Genetic-Bases-of-Metabolic-Diversity-with-mGWAS">Deciphering the Genetic Bases of Metabolic Diversity with mGWAS</h2>
<ul>
<li>过去，有大量的反向遗传学的研究来剖析代谢途径。
<ul>
<li><strong>从基因功能入手，解析代谢途径</strong></li>
<li>比如使用RNA-seq来发现某个基因表达差异，与某个代谢物相关。<strong>更加精确</strong></li>
<li>通过这种比较基因组分析和代谢数据相结合，来重新构建物种的相关代谢通路。</li>
</ul>
</li>
</ul>
<p>最近，物种内自然发生的代谢多样性和潜在的遗传决定因素吸引了越来越多的研究兴趣。某些代谢性状是由多个基因一起控制的，多基因遗传。特别是在植物中。</p>
<blockquote>
<p>比如：mQTL（metabolic Quantitative Trait Loci）映射，研究者们发现了VTE3基因的表观等位基因自然变异，这决定了番茄果实中维生素E的积累。</p>
</blockquote>
<ul>
<li>
<p><strong>mGWAS的应用扩展</strong>：</p>
<ul>
<li>
<p>为了识别<strong>自然群体中更广泛遗传变异对代谢多样性的影响</strong>，mGWAS最初在模式物种拟南芥Arabidopsis thaliana中应用，随后成功扩展到多个其他物种</p>
</li>
<li>
<p>这些研究涵盖了<strong>初级代谢物</strong>（如糖类、氨基酸等）和<strong>次级代谢物</strong>（如flavonoids等），涉及植物的营养组织（如叶片）和生殖组织（如花朵）。</p>
</li>
</ul>
</li>
<li>
<p><strong>农业性能与代谢物变异的不同遗传控制</strong>：</p>
<ul>
<li>以前的研究都表明，农业性状（如产量、抗病性等）由<strong>众多小效应位点</strong>控制，每个位点对性状的影响较小。
<ul>
<li>比如水稻的产量取决于几千个基因</li>
</ul>
</li>
<li>初级代谢物的含量通常由许多具有较小效应的位点决定。</li>
<li>但是，<strong>次级代谢物</strong>的<strong>自然变异</strong>通常由<strong>少数大效应位点</strong>控制，即每个位点对代谢物含量的影响较大。
<ul>
<li>比如某个植物的某个抗生素含量就跟少数几个基因有关，这个从传统的GWAS中就分析不出来</li>
</ul>
</li>
<li>此外，代谢遗传架构中存在决定大量代谢物变异的**“热点**”基因区域，这些区域受到自然选择的影响。
<ul>
<li>某些基因区域控制多个代谢物的生产，这些区域因为在抗病过程中起关键作用而在进化中被强化和保留。</li>
</ul>
</li>
</ul>
</li>
<li>
<p><strong>衍生性状</strong></p>
<ul>
<li>代谢物之间有相关性
<ul>
<li>在植物中，许多代谢物在相同或相关的代谢途径中被共同调控。它们可能经历相似的化学反应或修饰过程，甚至在不同的修饰方式之间竞争相同的底物。</li>
</ul>
</li>
<li>使用一些衍生性状
<ul>
<li>基于代谢物的绝对含量计算得出的，例如相关代谢物的总和或两个代谢物之间的比例。</li>
</ul>
</li>
<li>衍生性状和mGWAS
<ul>
<li>Angelovici等人通过检测氨基酸的绝对含量，识别出一个由多个相互连接的氨基酸组成的网络，并利用衍生性状（如氨基酸比例）进行GWAS，从而更有效地识别出影响这些氨基酸含量的基因。</li>
</ul>
</li>
</ul>
</li>
<li>
<p>此外还有 comparative GWAS，植物物种之间共享的“共同”基因座使用mGWAS，根据物种间代谢性状的共调控来识别候选基因。</p>
</li>
</ul>
<h1>Limitations and advantages of using metabolite-based genome-wide association studies: Focus on fruit quality traits</h1>
<p>2023年 《使用基于代谢物的全基因组关联研究的局限性和优势：关注水果品质性状》</p>
<p>过去通过<strong>Linkage</strong>分析了大量的物种中代谢物数量性状位点QTL。</p>
<p>现在下一代测序和高通量基因分型技术，mGWAS成为识别多基因农作物性状中遗传变异的有力工具。</p>
<p>在水果中，水果风味是香气挥发物和味道的复杂相互作用，是糖和酸比是风味接受度的关键参数。</p>
<p>这篇文章回顾了mGWAS技术在水果中与风味代谢物相关的精确基因多态性的研究进展。</p>
<ul>
<li>这篇文章自己的工作：
<ul>
<li><strong>柚子品种的mGWAS</strong>：在对194个柚子品种进行的mGWAS研究中，研究者们调查了成熟果实中单个初级代谢物和脂质代谢物的遗传控制。</li>
<li><strong>发现的关联</strong>：共识别出了14种初级代谢物（包括氨基酸、糖类和有机酸）的667个关联，以及47种脂质对应的768个关联。</li>
<li><strong>候选基因</strong>：通过这些关联，研究者们发现了与果实质量相关的重要代谢物（如糖类、有机酸和脂质）相关的候选基因，为进一步的功能验证和作物改良提供了依据。</li>
</ul>
</li>
</ul>
<h2 id="Introduction-2">Introduction</h2>
<p>遗传学的主要目标是研究基因型和表型之间的关系。尽管复杂性状的很大一部分变异是由于遗传变异造成的，但遗传变异和性状之间的机制通常知之甚少。</p>
<p>代谢物在植物的基因型和表型之间的关系中起着至关重要的作用。</p>
<p>培育精英品种的有效操作需要了解途径的遗传和生化基础以及控制这些性状的调节系统。</p>
<ul>
<li>代谢组学分析结合下一代测序技术
<ul>
<li>mQTL 方法一直是剖析农艺性状遗传基础的非常成功的工具，特别是研究营养品质的遗传决定因素</li>
<li>mGWAS</li>
</ul>
</li>
</ul>
<h2 id="2-限制mGWAS的分析因素">2 限制mGWAS的分析因素</h2>
<h3 id="2-5-GWAS-analysis-to-identify-metabolic-candidate-genes-underlying-fruit-quality-traits">2.5 GWAS analysis to identify metabolic candidate genes underlying fruit quality traits</h3>
<p>对于水果的性状来说，过育种者在过去几十年中主要通过选择颜色、硬度、外观和保鲜期等特性来改良水果的质量。这些选择标准虽然提升了水果的市场吸引力，但在这个过程中，水果的风味和香气性状却被忽视或减少了。</p>
<p>通过GWAS和代谢组的结合，就可以通过找到风味/香气相关的基因，然后从野生种中找到这些控制化合物合成的候选基因，引入到现代栽培品种中。</p>
<h3 id="2-6-Metabolomics-based-GWAS-in-citrus-fruits">2.6 Metabolomics-based GWAS in citrus fruits</h3>
<p>对于194种不同品种的柑橘品种，分析了150种化合物（55种初级代谢物和95种脂质）</p>
<p><img src="https://pic.imgdb.cn/item/676a81ebd0e0a243d4e94265.png" alt=""></p>
<h1>综述内容</h1>
<h2 id="框架">框架</h2>
<ul>
<li>
<p>第一篇Metabolic GWAS-based dissection of genetic bases underlying the diversity of plant metabolism</p>
</li>
<li>
<pre><code>The Development of GWAS
The Rise of Metabolic Diversity
Deciphering the Genetic Bases of Metabolic Diversity with mGWAS
mGWAS-Based Multi-dimensional Analysis
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line">- 第二篇Limitations and advantages of using metabolite-based genome-wide association studies: Focus on fruit quality traits</span><br><span class="line"></span><br></pre></td></tr></table></figure>
</code></pre>
</li>
</ul>
<p>Critical factors to consider in mGWAS analysis. General limitations<br>
Confounding factors in GWAS<br>
Allelic heterogeneity<br>
Low GWAS statistical power<br>
Rare causal variants<br>
GWAS analysis to identify metabolic candidate genes underlying fruit quality traits<br>
Metabolomics-based GWAS in citrus fruits</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line">- 第三篇Research Progress and Trends in Metabolomics of Fruit Trees</span><br><span class="line"></span><br><span class="line">   </span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>Metabolomics Strategies and Key Techniques<br>
Metabolomics Strategies  代谢组学策略<br>
Key Techniques in Metabolomics<br>
Data Processing  数据处理<br>
Application of Metabolomics in Fruit Trees</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line">- 第四篇 pdf</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>Plant metabolomics: an overview<br>
Current analytical techniques for plant<br>
metabolomics research<br>
Key steps and workfow for plant<br>
metabolome analysis<br>
Metabolomic studies in plant stress<br>
response</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line">**1. 基因组学与GWAS：代谢性状遗传基础的探索**</span><br><span class="line"></span><br><span class="line">参考内容：第一篇的 “The Development of GWAS” 和 “Deciphering the Genetic Bases of Metabolic Diversity with mGWAS”，第二篇的 “GWAS analysis to identify metabolic candidate genes underlying fruit quality traits”</span><br><span class="line"></span><br><span class="line">- 主要是第一篇</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>GWAS is an observational study of a genome-wide set of common genetic variants in experimental populations, to identify variants associated with a trait. Klein et al. successfully performed the first GWAS and identified a variant in the Complement Factor H gene strongly associated with age-related macular degeneration (Klein et al., 2005). In the last decade, GWAS has been proven to be powerful in dissecting the genetic basis for variation in complex phenotypes, including disease in humans and animals (Pickrell et al., 2016; Soldner et al., 2016; Duncan et al., 2017; Jin et al., 2017; Pasaniuc and Price, 2017; Visscher et al., 2017), and physiological and agricultural traits of plants (Huang et al., 2010; Tian et al., 2011; Horton et al., 2012; Chen et al., 2016; Yano et al., 2016; Xiao et al., 2017). However, population structure and unequal relatedness among individuals result in spurious associations and false discoveries in GWAS. Methodologically, much effort has been made to address population stratification (Devlin and Roeder, 1999; Abecasis et al., 2000; Liu et al., 2016). In mixed linear models (MLM), population structure is fitted as a fixed effect, whereas kinship among individuals is incorporated as the variance−covariance structure of the random effect for the individuals (Yu et al., 2006; Zhao et al., 2007). The MLM method has become the most popular method for GWAS because of its demonstrated effectiveness in correcting the inflation from many small genetic effects and controlling the bias of population stratification. Subsequently, a series of models based on MLM has been developed to more efficiently and accurately identify associations between variants and traits (Korte et al., 2012; Segura et al., 2012; Loh et al., 2015). Generally, MLM-based models are carried out with single-locus tests to identify associations between polymorphisms and traits. However, for complex traits determined by several large-effect loci, this approach may not be appropriate, especially in the presence of population structure. The application of multi-locus mixed models could lead to the identification of new causative loci with promising performance (Segura et al., 2012; Wen et al., 2017). Despite the contribution of single nucleotide polymorphism (SNP)-based GWAS to the dissection of genetic bases underlying complex traits in various species, SNP-based GWAS incurs a severe penalty for multi-testing due to the overlooked interactions between SNPs within a gene and weak signals aggregating within related SNP sets. To limit the false discovery rate associated with GWAS analyses, haplotype-based GWAS with high statistical power has been developed to identify causal haplotypes with specific combinations. Haplotype-based GWAS was able to identify new candidates for complex traits, although its power for detecting quantitative trait loci (QTLs) is much lower than that of SNP-based GWAS, especially for QTLs with low minor allele frequencies (Sato et al., 2016). Moreover, gene-based GWAS has also been developed, involving all variants within a gene (Yano et al., 2016; Zhang et al., 2016). The above-mentioned methods are based on the assumption that the phenotype or marker effect will follow a typically normal distribution. GWAS based on the Anderson−Darling (A−D) test is a complement to MLM-based GWAS methods, especially for complex quantitative traits determined by moderate effect loci or rare variants, and with abnormal phenotype distribution (Yang et al., 2014a). To refine the genomic localization of causal variants identified in GWAS, fine-mapping strategies have been developed by using statistical methods (Schaid et al., 2018).<br>
The essence of GWAS is to identify naturally occurring genetic variance associated with targeted traits. Although the method of GWAS is a means to discover candidate QTLs, the population used for GWAS essentially determines whether there are causative genes and variants underlying diversity in traits. For example, distinct genes have been identified to be responsible for natural variation in rice heading date in GWAS with different populations (Huang et al., 2010; Yang et al., 2014b; Huang et al., 2015; Yano et al., 2016). Recently, kinds of genetic populations have been used in GWAS to efficiently identify causative genes for agricultural traits. For instance, approximately 1500 diverse varieties of hybrid rice were used to elucidate the genetic basis of rice heterosis (Huang et al., 2015). In addition to the natural populations used in GWAS, well designed artificial populations have been developed to efficiently identify genetic variants determining complex traits. For example, a nested association mapping populations (NAM) of maize, derived from crosses between a reference line and other founder inbreds, have been used in GWAS of developmental traits and resistance to pathogens with improved mapping resolution (Kump et al., 2011; Poland et al., 2011; Tian et al., 2011; Li et al., 2016; Wu et al., 2016). However, the extremely unbalanced parental compositions might dilute the GWAS efficiency (Xiao et al., 2017). To thoroughly intercross and effectively balance the contributions of all founder lines, the advanced generation intercross (MAGIC) populations of rice were established in various species (Kover et al., 2009; Huang et al., 2012; Bandillo et al., 2013; Dell’Acqua et al., 2015; Islam et al., 2016). The application of NAM or MAGIC populations is largely limited, due to the requirement for extensive field and laboratory effort. The random-open-parent association mapping (ROAM) population, consisting of different existing RIL populations, improves genetic resolution and statistical power to identify variants with minor effect and low frequency (Giraud et al., 2014; Xiao et al., 2016).<br>
Over the last decades, numerous reverse genetic studies have been carried out to dissect metabolic pathways (Zhao et al., 2016; Perchat et al., 2018; Tian et al., 2018). Meanwhile, genes determining the accumulation of compounds with vital biological functions have also been characterized in studies of phenotypic traits (Cardoso et al., 2014; Yang et al., 2017; Arisz et al., 2018; Song et al., 2018). Recently, improved sequencing technologies have provided new chances to dissect metabolic pathways. For instance, Capuri et al. identified two missing enzymes in the biosynthesis of vinblastine, using RNA-seq data (Caputi et al., 2018). Furthermore, with the increasing number of genome sequences available from closely or more distantly related species, it is becoming possible to combine comparative genomic analysis with metabolic data in gene identification and pathway elucidation (Denoeud et al., 2014; Huang et al., 2017). To dissect the mechanism of rubber production and species adaptation, Tang et al. performed a comparative genomic analysis, using genome and transcriptome sequencing with six cultivars of rubber trees. They characterized the crucial role of a striking expansion of the REF/SRPP (rubber elongation factor/small rubber particle protein) gene family and its divergence into several laticifer-specific isoforms in rubber biosynthesis. Moreover, the stringent control of ethylene synthesis under active ethylene signalling and response in laticifers resolved a longstanding mystery of ethylene stimulation in rubber production (Tang et al., 2016). Recently, the naturally occurring metabolic diversity within species and the underlying genetic determinants have attracted increasing research interests (Luo, 2015). Studies with natural or artificial populations in several species discovered that plant metabolism is moderately inheritable and shows polygenic inheritance (Chen et al., 2014; Alseekh et al., 2015; Soltis and Kliebenstein, 2015). Metabolites displayed high heritability and constant diversity within experimental populations, making the metabolome of plants an ideal target trait for population genetics studies. There are legions of examples of population genetic studies on the plant metabolome, for example, mQTL mapping with molecular markers identified natural variation in epialleles of VTE3, determining vitamin E accumulation in tomato fruits (Quadrana et al., 2014). Ultra high-density genetic maps have provided fresh impetus toward the identification of tremendous numbers of genes underlying metabolic diversity between parents (Gong et al., 2013; Wen et al., 2015).<br>
To identify effects of more widespread genetic variants on metabolic diversity across natural populations, mGWAS was initially applied in the model species Arabidopsis thaliana and then successfully performed in and extended to a number of other species, especially important crops. These studies covered primary metabolites (Strauch et al., 2015; Angelovici et al., 2016; Du et al., 2018) and secondary metabolites (Dong et al., 2015; Peng et al., 2016, 2017), ranging from vegetative tissue (Chen et al., 2014; Matsuda et al., 2015; Fang et al., 2016) to reproductive tissue (Wen et al., 2014; Zhou et al., 2016; Zhu et al., 2018). Generally, agricultural performance is under the control of numerous loci of small effects (Huang et al., 2010). However, natural variation in secondary metabolites tends to be controlled by a small number of loci with large effects (Chen et al., 2014), while the contents of primary metabolites are generally determined by many loci of smaller effects (Rowe et al., 2008; Chan et al., 2010; Wen et al., 2015). In addition, a common feature in the genetic architecture of metabolism is the presence of ‘hotspots’ of major genes/genome regions determining the natural variation in large sets of metabolites (Chan et al., 2010; Knoch et al., 2017). More detailed evaluation revealed that some of the hotspots in Arabidopsis are within the regions of the genome previously identified as being subject to recent strong positive selection (selective sweeps) and regions showing trans-linkage to these putative sweeps, suggesting that selective forces have impacted genome-wide control of Arabidopsis metabolism (Chan et al., 2010). Causative genes identified in GWAS of complex traits tend to be regulatory genes, such as transcriptional factors (Huang et al., 2010; Yano et al., 2016). However, most of the causative genes for metabolic diversity encode enzymes involved in the production of targeted compounds (Chen et al., 2014; Wen et al., 2014; Matsuda et al., 2015; Wen et al., 2015; Chen et al., 2016). Hence, it is much easier for researchers to predict gene candidates in mGWAS. Although crucial roles of several transcriptional factors have been revealed in mGWASs (Okada et al., 2009; Yamamura et al., 2015; Zhu et al., 2018), the genetic bases for the diversity in regulation of metabolite accumulation remain to be unveiled. In addition, to date, mGWAS strategy is applied in limited species, with well sequenced populations available (Chen et al., 2014; Wen et al., 2014; Angelovici et al., 2016). Recently, a cost-effective approach for low-coverage sequencing-based GWAS has been developed (Wang et al., 2016). In such a strategy, low-coverage sequence data combined with imputation-based methods were used for genotype calling, which provides easy resolution of haplotypes and higher genotype calling accuracy (Wang et al., 2016). This strategy will be helpful for further GWAS on both metabolic and phenotypic traits in special species, such as medicinal plants.<br>
In contrast with the weak linkage among agricultural traits, tremendous numbers of metabolites are tightly correlated due to the co-regulation of natural products in the same pathway, the related reactions in metabolic pathways, the same modification of structurally similar compounds, and the competition for substrates of different modifications. Aschard et al. found that principal-component analysis (PCA)-based approaches could maximize the power of GWAS on correlated phenotypes (Aschard et al., 2014). Similar strategies may also be helpful for dissection of genetic bases for diversity in correlated compounds. In addition, derived traits generated from absolute levels of metabolites provided new insights into a metabolic network, including the sum of related metabolites or the ratio of two related compounds (Sauer et al., 1999; Weckwerth et al., 2004; Wentzell et al., 2007). Exploring the association of metabolites could also contribute to the dissection of metabolic pathways with mGWAS. For instance, Angelovici et al. detected the absolute content of 18 free amino acids and identified a network of 12 connected amino acids. Derived traits were calculated as a ratio of connected metabolic pairs or as a ratio of a single amino acid, to its fully or partially connected metabolic group (Angelovici et al., 2016). Network-guided GWAS based on the derived traits improved identification of causative genetic determinants affecting free amino acids. The requirement of the absolute content of metabolites may limit the application of network-guided GWAS, especially for GWAS based on metabolomic data.<br>
Over the last decades, fruit quality traits defined as flavor, aroma, mount-feeling, and postharvest properties have been selected for breeders by considering a diverse of properties such as color, firmness, appearance or shelf life. This historical selection has result in a genetic erosion of flavor/aroma traits in modern cultivars, therefore a loss of variants controlling fruit quality directly associated to metabolite level (Giovannoni, 2018). Thus, in general, the current modern cultivars lack in compounds that impact on those fruit quality traits. This dilution of flavor/aroma compounds should be correctable by restoring or introducing specific metabolites (desirable trait) from old cultivars or wild species to modern commercial cultivars. As described above, the combination of GWAS analysis with metabolomics have been successfully used to discover genes underlying fruit metabolite content of flavor/aroma impact compounds. Once the candidate genes controlling the synthesis of these compounds, their superior alleles can be re-introduced in modern cultivars.</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line">**2. 代谢组学：技术、策略与应用**</span><br><span class="line"></span><br><span class="line">参考内容：第三篇的 “Metabolomics Strategies and Key Techniques”，第四篇的 “Current analytical techniques for plant metabolomics research”</span><br><span class="line"></span><br><span class="line">主要是第三篇</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>Metabolomics Strategies<br>
Metabolites in plants can generally be divided into primary and secondary metabolites. Primary metabolites are the basic material conditions necessary for the growth and development of organisms, while secondary metabolites are metabolites produced only at a certain stage or period of growth of an organism (Koch, 2004; Carreno-Quintero et al., 2013). Depending on the purpose of the experiment, researchers can choose individual metabolomics strategies, including targeted metabolomics, non-targeted metabolomics, pseudo-targeted metabolomics, and widely targeted metabolomics.</p>
<p>Targeted metabolomics assays are performed on known compounds with targeted extraction methods and high-purity standards to quantify the target metabolism. Generally, targeted metabolomics focuses on fewer compounds and can be searched directly in classified databases, for example, databases for glycans and lipids such as LipidMaps and LipidBank (Cajka and Fiehn, 2016). Non-targeted metabolomics assays are performed on unknown compounds. It covers as many compounds of different properties and classes as possible in the extraction and detection, with a relatively broad coverage of substances. However, both the qualitative and quantitative properties of compounds are compromised by the lack of standards. Generally, non-targeted metabolomics studies are used to screen for differential metabolites in different treatments, and targeted metabolomics studies are used for the next more precise study.</p>
<p>To overcome the disadvantages of non-targeted metabolomics, widely targeted metabolomics, also known as second-generation targeted metabolomics, allows for the simultaneous detection of thousands of substances by the establishment of a database of thousands of metabolite specimens (Wei et al., 2013). Pseudo-targeted metabolomics is a quantitative ion selection algorithm based on untargeted metabolomics. It is used to perform quantitative ion selection for all detected metabolites. The ion-pair information of metabolites is obtained by high-resolution mass spectrometry (HR-MS), and the abundance of metabolites is measured by targeted multiple reaction monitoring (MRM). The method has high coverage, good linearity, and reproducibility and does not require standards to limit the metabolites’ detection. Both known and unknown metabolites in the sample can be measured (Zheng et al., 2020).</p>
<p>Spatial metabolomics integrates traditional metabolomics technology and MS imaging technology, which can not only identify what substances are contained in a sample and their contents as traditional metabolomics but also detect the spatial distribution information of compounds in a single experiment to achieve qualitative and quantitative positioning at the same time (Li et al., 2018).<br>
a<br>
Key Techniques in Metabolomics<br>
The exploration of the diversity of fruit tree metabolism and potential molecular mechanisms by which fruit tree cells control their own chemical composition depends on the advances of analytical methods.</p>
<p>Sample Processing<br>
Samples should be collected representatively, with priority given to normally developing plants. Furthermore, 3–6 individuals should be selected as a source of biological replicates to avoid differences due to excessive individual differences. Ice boxes or ice packs should be used to collect samples in the field to reduce the degradation of metabolites by enzymes that are still active after sample collection. At the same time, samples should be collected with certain accuracy, and the samples of the experimental and control groups should be consistent in terms of time (Urbanczyk-Wochniak et al., 2005), site, and processing conditions, as far as the experiment allows. If the kernels are not studied, it should be removed with a scalpel in a low-temperature metal bath on an RNAase reagent-treated bench. For the study of the peel, the pulp should be taken as less as possible (Montefiori et al., 2009; de Godoy et al., 2013; Rudell et al., 2017). Freezing of the whole fruit should be avoided if possible during the sampling process, which will make the subsequent freeze-drying and grinding work difficult, and once the whole fruit is freeze-thawed, the whole experimental results will be affected. Then, the sample will be stored in a suitable volume of a centrifuge tube, marked and quickly put into liquid nitrogen for precooling, and then put into −80°C refrigerator for storage. Note that the samples should not be stored in tin foil, self-sealing bags, Ziploc bags, plastic bags, etc.; such packaging is easy to break under low temperature or in the process of transportation and easy to make the markings blurred, resulting in sample confusion.</p>
<p>The metabolites are usually extracted separately with water or organic solvents (e.g., methanol, chloroform, etc.) to obtain aqueous and organic solvent extracts, thus separating the non-polar lipophilic phase from the polar phase. To enrich the concentration of metabolites during the study, the process of vacuum concentration and spin-drying of the extracts before redissolution is often added. In the process of sample analysis method establishment, the optimal method can be figured out by trying different combination of extraction solvents, extraction temperatures, ratios of sample and solvent, redissolved solvent, mobile phase gradients, and mobile phases and by the controlled variable method (Kim and Verpoorte, 2010). The detection of non-targeted metabolites should include as many compound peaks as possible, and the detection of targeted metabolites should ensure that all concerned metabolites can be detected. The sample sequence requires the insertion of quality control samples at intervals of 10 or 20 during the detection by liquid chromatography-MS (LC-MS), which generally includes all compounds of the sample being tested or a mixture of a representative batch of samples. In addition, compounds not contained in the fruit trees are added as internal standards during the extraction of the samples to calibrate the loss of sample compounds during the extraction and detection process and thus to correct the peak areas of other compounds. In vivo direct-immersion solid-phase microextraction followed by gas chromatography-time-of-flight mass spectrometry method (SPME GC × GC-TOF-MS) can detect volatile compounds directly without extracting the compounds (Risticevic et al., 2016, 2020).<br>
Analysis<br>
Chromatography is used for the purpose of separating metabolites according to the different affinities of compounds and columns and is generally divided into LC and gas chromatography (GC). LC is used to detect non-volatile compounds, polar compounds, thermally unstable compounds, and high-molecular-weight compounds (including proteins, peptides, and polymers). The mobile phase generally uses methanol, acetonitrile, water, or isopropanol; the added acid is generally acetic acid or formic acid, inorganic acid cannot be used (inorganic acid may corrode the pipeline); the added base is generally ammonium hydroxide or ammonia, generally does not use metal bases (will corrode the pipeline). Trifluoroacetic acid (TFA) helps to improve the separation of the liquid phase but will produce ion suppression in the positive and negative ion modes of MS. Triethylamine/trimethylamine (TEA/TMA) contributes to the formation of negative ions. GC can be used for the detection of volatile compounds, the mobile phase is an inert gas, and samples with a certain vapor pressure and good stability at column temperature can be detected. Substances with low volatility and easily decomposed by heat can be transformed by derivatization into derivatives with high volatility and good thermal stability for detection. N-methyl-N-trimethylsilyl trifluoroacetamide (MSTFA) and methoxamine hydrochloride are often used as derivatization reagents, as preliminary studies have shown that these compounds are the most suitable for the derivatization of plant metabolites (Fiehn et al., 2000).</p>
<p>Mass spectrometry separates some of the ions based on the mass to charge ratio. The ions pass through the analyzer and are separated according to different mass to charge ratios. Ions with the same mass to charge ratio are clustered together to form a mass spectrogram. Commonly used mass spectrometers are triple quadrupole mass analyzer (Yost and Enke, 1978), time-of-flight mass analyzer, ion trap mass analyzer, and electrostatic field orbital trap. The triple quadrupole mass analyzer consists of four straight rod electrodes suspended equidistantly parallel to the axis. Under the action of a certain DC/VC, ions with m/z meeting certain requirements can pass through the quadrupole to the detector, while other ions are filtered out. The advantages of the triple quadrupole are light weight, small size, and low cost. The time-of-flight mass analyzer uses pulses to extract ions from an ion source. The ions are accelerated by an accelerating voltage, have the same kinetic energy, and enter the drift tube. Ions with a small mass to charge ratio are the fastest and reach the detector first. Larger ions reach the detector last. The advantages of TOF MS are a wide range of mass to charge ratio of the detected ions, high sensitivity, and suitability for secondary tandem MS. A fast scanning speed is suitable for studying very fast processes. The principle of an ion trap mass analyzer is similar to that of quadrupole mass analyzer. When the HF voltage amplitude and HF voltage frequency are fixed to a certain value, only ions with a certain mass to charge ratio can be stabilized on a certain track in the trap. By rotating and changing the electrode voltage, different m/z ions fly out of the trap and reach the detector. The advantage of ion trap mass spectrometer is that a single ion trap can realize multistage “time” tandem MS, simple structure, low price, high cost performance, high sensitivity, and large mass range. The electrostatic field ion orbitrap is where analytes are ionized in an ion source and then sequentially enter a quadrupole, a C-trap, and an Orbitrap. The electrostatic field ion orbitrap mass spectrometer can perform high precision mass scans, and it is more stable than other mass analyzers.</p>
<p>Mass spectrometry characterizes compounds by different characteristic ions of different compounds. Chromatographic methods enhance the coverage of metabolites and improve the quantitative accuracy of MS. The most commonly used separation and analysis methods are LC-MS (Vos et al., 2007), GC-MS (Lisec et al., 2015). For LC-MS and GC-MS methods, internal standards are added before sample processing to correct errors caused by sample pretreatment and instrument response.</p>
<p>Data Processing<br>
Data processing in metabolomics generally consists of two major steps, namely, data preprocessing and statistical analysis (Ma and Qi, 2021). Data preprocessing includes steps such as removing system noise signals, removing disturbing signals caused by system instability, and removing operational errors to provide a more reliable dataset for the next statistical analysis. Prior to formal data processing, tools such as XCMS, MZmine, and MarkerView are used to process the original peak area and other data for non-targeted metabolomics data analysis. The first step of the statistical analysis is unsupervised multivariate statistical analysis, usually using the principal component analysis (PCA). The second step may be the univariate statistical analysis to screen for variables with statistically significant differences in different groups. The third step is a supervised multivariate statistical analysis such as partial least squares discriminant analysis (PLS-DA) to select the variables that contribute more to the classification of the sample, i.e., screen for markers.</p>
<p>The purpose of the PCA analysis (Shaffer, 2002) is to find specific values that produce differences in a large amount of sample data. PLS-DA analysis will classify samples into groups, which is good for finding the similarities and differences between sample groups; orthogonal partial least squares discriminant analysis (OPLS-DA) can also distinguish the group differences between samples, but this analysis can better focus on some positive correlation data. The results of data analysis are also subject to the t-test and variable importance in projection (VIP) ranking values to screen for differential metabolites. It is generally accepted that variables meeting both P &lt; 0.05 and VIP &gt; 1.0 are differential metabolites. Metabolic pathways can be resolved using databases such as HMDB, KEGG, Reactome, BioCyc, and MetaCyc.</p>
<p>Application of Metabolomics in Fruit Trees<br>
Fruit tree metabolomics has developed rapidly in recent years, and many important research results have been achieved in combination with transcriptomics, genomics, proteomics, quantitative trait locus (QTL), and genome-wide association study (GWAS) (Tables 2, 3). These research results mainly focus on the mechanism of fruit quality formation, metabolite markers of special quality or physiological period, key genes and metabolites of fruit tree resistance to biotic/abiotic stress, the influence of environment on the fruit tree, and fruit tree population genetic basis by combining with QTL and GWAS to narrow down QTL regions, predict candidate genes, construct the regulatory network, etc. (Figure 3).<br>
Metabolomics has developed as an outstanding scientifc<br>
feld; however, a single analytical technique is not adequate<br>
to detect and quantify the metabolites (Weckwerth, 2003;<br>
Templer et al. 2017). Presently, various metabolomic techniques are being applied in plant metabolomics research, as<br>
discussed in the introduction. Out of these, GC, MS, NMR,<br>
and HPLC dominate the metabolite tools. Two basic techniques, MS and NMR, are used in modern metabolomics,<br>
including the generation of metabolomics data. Interestingly,<br>
NMR is preferred to MS because of its high capacity in<br>
detecting protein binding sites, direct binding of target proteins, physical properties of ligands, and uncovered protein<br>
structure coupled with ligands. Metabolite exposure reliant on NMR uses magnetic properties of various nuclei of<br>
atoms. The diferent applications, such as metabolite profling and fngerprinting, metabolic fux, and atomic structural<br>
details of diferent biological samples, are integrated with<br>
NMR. Owing to the non-destructive nature of NMR with a<br>
smaller molecular weight is widely used to detect metabolites (Eisenreich and Bacher 2007; Kim et al. 2010). Hence,<br>
this technique is so sensitive, and it has a low abundance of<br>
biomarkers that causes its limited use. Except for NMR, the<br>
MS technique has the best sensitivity, and researchers can<br>
get an extensive range of metabolome data. This technique<br>
would help researchers to detect molecules and metabolic<br>
biomarkers that can rebuild metabolic networks and pathways. Diferent ionization methods such as matrix-assisted<br>
laser desorption/ionization (MALDI-TOF), atmospheric<br>
pressure chemical ionization (APCI), and electrospray ionization (ESI) were accurately detected by MS (Issaq et al.<br>
2009). To get accurate results, MS is coupled with various<br>
techniques such as feld asymmetric waveform ion mobility<br>
spectrometry (FAIMS), CE, GC, FT-ICR, and LC. Figure 2<br>
indicates the comparison of frequently working analytical<br>
techniques in plant metabolomics research.<br>
Notably, MS has obtained a progressively vital role in<br>
the feld of metabolomics and proteomics due to the signifcant progress that has been made in instrument technologies. The frequently used technique for untargeted analysis<br>
is GC–MS (Rohlof 2015). Sample derivatization was done<br>
by the GC–MS technique, making the compound volatile;<br>
however, some compounds are left as underivatized during<br>
analysis. GC–MS has been recognized as a high-throughput analytical technology with a high rate of sensitivity for<br>
metabolic profling. GCxGC-TOF–MS enhanced the output through the segregation of co-eluting peaks (Hurtado<br>
et al. 2017). Higher mass primary and specialized metabolites (&lt;1500 Da) are detected by targeted and untargeted<br>
techniques facilitated by LC–MS that uses ESI and APCI<br>
(Turner et al. 2016). Identifcation of several metabolites<br>
increases peak resolution, and mass accuracy was done in<br>
a short time with the help of UPLC coupled with QTOFMS (Chawla and Ranjan 2016). In targeted and untargeted<br>
metabolomics analysis, high-resolution separation of metabolites is mainly done by CE-MS (Ramautar et al. 2019).<br>
FT-ICR-MS is driven by high-resolution mass analysis that<br>
provides extensive and reliable detection of metabolites. It<br>
is also coupled with separation techniques to settle complex<br>
matrices, and ion separation was also done by this technique<br>
(Ghaste et al. 2016; Nakabayashi et al. 2016; Lopes et al.<br>
2017).<br>
Data produced from the above-mentioned techniques are<br>
processed by Met-Align, PlantMAT, MET-XAlign, METCOFEA, XCMS, and ChromaTOF, etc. (Table 1). Statistical<br>
analysis of identifed metabolites is followed by using a combination of (1) univariate analysis (Student t test; ANOVA;<br>
Mann–Whitney U test; Benjamini–Hochberg false discovery<br>
rate correction; Kruskal Wallis 44%), and (2) multivariate<br>
analysis (principal component analysis (PCA); partial least<br>
squares discriminant analysis (PLS-DA); orthogonal partial least squares (O-PLS); high-content screening (HCA);<br>
heatmap, correlation analysis, neural networks, genetic algorithms, and random forest methods. Currently, several diferent software and online tools are available for metabolome<br>
analysis, like MetaboAnalyst, MetaboliteDetector, MetaMapR, MetExplore, Cytoscape, g:Profler, Gene-set enrichment analysis (GSEA), Metabolite-set enrichment analysis (MSEA), EnrichmentMAP, Workfow4Metabolomics</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line"></span><br><span class="line"></span><br><span class="line">**3. 基因组学、代谢组学与GWAS的结合：多组学分析的前沿探索**</span><br><span class="line"></span><br><span class="line">参考内容：第一篇的 “mGWAS-Based Multi-dimensional Analysis”，第二篇的 “Metabolomics-based GWAS in citrus fruits”</span><br><span class="line"></span><br><span class="line">- 第一篇</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>Even though numerous key genes with large effects on the accumulation of metabolites have been characterized with mGWAS, single data type analyses may lead to missing or unreliable information. Moreover, the extent of linkage disequilibrium (LD) in plants often ranges over several hundred kilobases, especially in self-pollinating crops such as rice (Mather et al., 2007), resulting in the inclusion of many candidate genes in a single LD block. Hence, multi-dimensional data analysis is essential for enhancing our understanding of metabolic diversity and its corresponding genetic basis. Multi-dimensional data could be collected from different tissues or conditions, various analysis strategies, and/or multi-omics data.</p>
<p>As we have discussed above, the spatial−temporal and environment-dependent expression of key regulators of metabolic pathways lead to the metabolic diversity at the individual level. Some studies based on metabolome data from different tissues (Chen et al., 2014; Matsuda et al., 2015; Chen et al., 2016) or under different conditions (Wu et al., 2018) have also identified diverse genetic determinants for the same or structurally similar compounds. Although combined mGWAS under different conditions enhanced the power to identify causative genes (Wu et al., 2018), knowledge about the variation in the content ratio of each individual metabolite is still lacking, as well as knowledge about deciphering its genetic basis with mGWAS. However, rigorously designed mGWAS based on metabolite content ratios under different conditions will contribute not only to the understanding of metabolic diversity under constantly changing environments, but also lead to the identification of key regulators for stress responses from the lens of the metabolome.</p>
<p>With the development of genome sequencing and bioinformatics technologies, various strategies have been improved to decipher the genetic diversity of targeted traits, such as association and linkage mapping. Association mapping by GWAS for plants has provided new genetic and biochemical insights into metabolomes. While it is true that GWAS benefits from abundant genetic variants allowing the location of QTL to be inferred with a high resolution, the inherent population structure and presence of rare variants in experimental populations reduce the statistical power of GWAS (Huang et al., 2010). Linkage mapping using artificial populations such as RILs and ILs is likely to be more powerful in identifying alleles with low frequency or small effects in the population (Fridman et al., 2004; Schauer et al., 2006). Hence, combined linkage and association mapping are powerful not only in cross-validating results from one another but also in identifying new causative loci (Luo, 2015). Recent work by Peng et al. provides encouraging evidence for the assumption that joint-linkage and association mapping are powerful not only in cross-validating results from one another but also in complementing each other in identifying new causative loci (Peng et al., 2016). However, association mapping and linkage mapping in this work were performed with distinct populations. Newly designed multi-parent mapping populations, such as NAM, MAGIC, and ROAM, are suitable for joint-linkage and association mapping (Xiao et al., 2017).</p>
<p>Multi-dimensional analyses have been increasingly used to provide clues for understanding biological mechanisms, since combining multiple datasets can compensate for missing or unreliable information in any single data type (Ritchie et al., 2015). Combined analyses with metabolic and transcriptomic data have been utilized to dissect various metabolic pathways (Itkin et al., 2013; Cardenas et al., 2016). Taking advantage of available experimental populations, transcriptomic data were collected from 240 lettuce accessions sampled from the major horticultural types and wild relatives. Nine eQTLs were identified in GWAS to regulate genes associated with flavonoid biosynthesis (Zhang et al., 2017). Multi-dimensional analysis of genomes, transcriptomes, and metabolomes could be more powerful to provide new insights into metabolic diversity, using large population collections. For instance, Zhu et al. performed a multi-dimensional analysis with a large collection of tomato accessions. More than 13 000 triple relationships (metabolite−SNP−gene) were identified in the overlap of the mGWAS and eQTL results. The combined analysis in this work led to the identification of key determinants for metabolic diversity in the tomato but also provided novel insights into the comprehension of the effect of domestication and breeding on metabolic diversity (Zhu et al., 2018). Our understanding on the genetic bases underlying diversity of targeted traits is enhanced at a considerable larger scale using multi-dimensional analysis with metabolome data (Zhu et al., 2018), compared to that with phenotypic data (Wang et al., 2015; Wilson et al., 2015). This is beneficial from both high-throughput and accurate evaluation of metabolite contents and tremendous variation in content of a large proportion of detectable metabolites.</p>
<p>As a counterpart of GWAS from the lens of epigenomics, epigenome-wide association studies (EWAS) have been developed and have provided new insights to epigenetically controlled diseases in human (Rakyan et al., 2011). To explore the effects of DNA methylation on human metabolism, EWAS was conducted with DNA methylation data and metabolic data from human blood (Petersen et al., 2014). The contribution of epigenetic variation to variation in phenotypic traits has been studied with populations of epigenetic recombinant inbred lines in Arabidopsis (Johannes et al., 2009; Reinders et al., 2009). However, the important and potentially independent roles of epigenetic variants played in plant metabolic diversity remain to be illustrated, which may be an interesting topic for combined analyses of EWAS and metabolomics studies.</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line"></span><br><span class="line">- 第二篇</span><br><span class="line"></span><br></pre></td></tr></table></figure>
<p>Here, we tested the effectiveness of EMMA method by conducting the first metabolomics-based GWAS in citrus fruits. Considering the large variation in metabolism within species (Schauer et al., 2004, Bernillon et al., 2012, Wahyuni et al., 2012, Vallarino et al., 2018), we generally expect to easily identify genetic variants controlling metabolite levels. In this study, we assayed 150 biochemical compounds (55 primary metabolites and 95 lipids; Table S2-S3) in ripe fruit for a set of 194 diverse Citrus grandis accessions (Table S4; Fig. S1). Using Bonferroni correction based in the effective numbers of independent markers, we significantly detected 667 associations for a total of 14 primary metabolites including amino acids, sugars, and organic acids, and 768 association corresponding to 47 lipids (Table S5-S6 and Fig. 1; Manhattan; p ≤ XX (LOD ≥ 6.0<br>
In general, from the perspective of genomic localization, we observed that the detected metabolite-SNP associations were spread across the genome (Fig. 1). However, for primary metabolites, a single metabolite mapped to multiple loci (ranking from 8 to 162 associations for individual primary metabolite at LOD ≥ 6.0). For sugars and organic acids, we observed cluster of associations that co-located on Chr 1, 3, and 5 as have been reported in other plant species (Lerceteau-Köhler et al., 2012; Castro, Lewers (2016). This result indicates that in these genomic regions, common genes might be involved in biosynthesis of different sugars and organic acids. Moreover, these observations can be attributed to polygenic regulation of primary metabolism rather than a few major genes as previously suggested for Arabidopsis (Wu et al., 2016) and citrus for the sugar/acid ratio (Imai et al., 2018) and also in agreement with other studies (Keurentjes et al., 2006, Chan et al., 2010). In our study, associations for lipids were also identified in several loci, however we also observed some major large-effect loci in agreement with previous studies (Chan et al., 2011, Wu et al., 2018a), indicating that the regulatory network of lipid metabolism are more linear in comparison to more complex regulation of primary metabolism.<br>
In order to discover novel and underexplored candidate associations, we searched into the most likely candidate genes of the identified loci that were significantly associated with high LOD value based on (i) gene annotation, and (ii) prior knowledge. Thus, we focused on loci in regions showing high significant association (LOD ≥ 8.0) where one or a few candidate genes could be identified near the peak SNPs. Surprisingly, a relative lower number of significant SNPs associated to metabolites in comparison to other studies performed in other species using the same routine association mapping analysis was detected (Ruggieri et al., 2014, Wu et al., 2016, Wen et al., 2018). Here, the lower statistical power can possible due to population size and/or population structure. It is widely accepted that many false negatives for co-founded traits are correlated with population structure (Yang et al., 2013, Xing et al., 2015, Fan et al., 2006b, Mao et al., 2010). This result raises the possibility that the analyzed Citrus grandis collection presents some structural pattern that limit the potential to detect significant associations. Recently, analysis of the genome sequences of 130 accessions of different citrus species had helped to map the evolution origin and evolutionary changes in Citrus genus during domestication (Wu et al., 2018b). They found that citrus fruit crops had a complex history of admixture for which the current diversity is linked to widespread ancient introgression (Wu et al., 2018b). Interestingly, Wang et al., (2018) using deep genome-sequencing 104 citrus accessions provided concrete genetic evidence about its genetic relationships and domestication trajectories. In particular, the genetic structure of 46 mandarin wild and cultivated accessions, revealed that two Mangshan wild mandarin accessions are the wild progenitors of two independent domestication events resulting in two groups of cultivated mandarins. Interestingly, all cultivated mandarins and few wild mandarins exhibited interspecific introgression signatures from another citrus species, although the wild mandarins had purer genetic backgrounds that cultivated accessions. It is also believed that this interspecific hybridation play special roles in phenotypical diversity of this fruit, particularly fruit size and acidity (Wu et al., 2018b, Wang et al., 2018).</p>
<pre><code>
</code></pre>
</article><div class="post-copyright"><div class="post-copyright__author"><span class="post-copyright-meta"><i class="fas fa-circle-user fa-fw"></i>文章作者: </span><span class="post-copyright-info"><a href="http://example.com">wanzi</a></span></div><div class="post-copyright__type"><span class="post-copyright-meta"><i class="fas fa-square-arrow-up-right fa-fw"></i>文章链接: </span><span class="post-copyright-info"><a href="http://example.com/2024/12/24/[%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB]%E5%9F%BA%E5%9B%A0%E7%BB%84-%E4%BB%A3%E8%B0%A2%E7%BB%84mGWAS%E7%BB%BC%E8%BF%B0/">http://example.com/2024/12/24/[%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB]%E5%9F%BA%E5%9B%A0%E7%BB%84-%E4%BB%A3%E8%B0%A2%E7%BB%84mGWAS%E7%BB%BC%E8%BF%B0/</a></span></div><div class="post-copyright__notice"><span class="post-copyright-meta"><i class="fas fa-circle-exclamation fa-fw"></i>版权声明: </span><span class="post-copyright-info">本博客所有文章除特别声明外，均采用 <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a> 许可协议。转载请注明来自 <a href="http://example.com" target="_blank">丸子Blog</a>！</span></div></div><div class="tag_share"><div class="post-meta__tag-list"><a class="post-meta__tags" href="/tags/%E5%A4%9A%E7%BB%84%E5%AD%A6/">多组学</a></div><div class="post_share"><div class="social-share" data-image="/img/cover6.jpg" data-sites="facebook,twitter,wechat,weibo,qq"></div><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/butterfly-extsrc@1.1.3/sharejs/dist/css/share.min.css" media="print" onload="this.media='all'"><script src="https://cdn.jsdelivr.net/npm/butterfly-extsrc@1.1.3/sharejs/dist/js/social-share.min.js" defer></script></div></div><nav class="pagination-post" id="pagination"><div class="prev-post pull-left"><a href="/2025/01/05/%5B%E7%A7%91%E7%A0%94%E5%B7%A5%E5%85%B7%5DAI%E7%94%BB%E5%9B%BE%E4%BD%BF%E7%94%A8/" title="AI画图学习"><img class="cover" src="/img/cover19.jpg" onerror="onerror=null;src='/img/404.jpg'" alt="cover of previous post"><div class="pagination-info"><div class="label">上一篇</div><div class="prev_info">AI画图学习</div></div></a></div><div class="next-post pull-right"><a href="/2024/12/22/%5B%E7%94%9F%E4%BF%A1%E5%AF%BC%5Dclass8_KEGG%E3%80%81GO%E6%95%B0%E6%8D%AE%E5%BA%93/" title="生信Class8_KEGG、GO数据库"><img class="cover" src="/img/cover2.jpg" onerror="onerror=null;src='/img/404.jpg'" alt="cover of next post"><div class="pagination-info"><div class="label">下一篇</div><div class="next_info">生信Class8_KEGG、GO数据库</div></div></a></div></nav><div class="relatedPosts"><div class="headline"><i class="fas fa-thumbs-up fa-fw"></i><span>相关推荐</span></div><div class="relatedPosts-list"><div><a href="/2024/12/04/%5B%E7%94%9F%E4%BF%A1%E7%AC%94%E8%AE%B0%5DGWAS%E5%92%8CLinkAnalysis/" title="GWAS和LinkAnalysis"><img class="cover" src="/img/cover11.jpg" alt="cover"><div class="content is-center"><div class="date"><i class="far fa-calendar-alt fa-fw"></i> 2024-12-04</div><div class="title">GWAS和LinkAnalysis</div></div></a></div><div><a href="/2024/07/20/%5B%E7%94%9F%E4%BF%A1%E7%AC%94%E8%AE%B0%5D%E8%A1%A8%E8%A7%82%E5%9F%BA%E5%9B%A0%E7%BB%84%E5%AD%A6/" title="表观基因组学"><img class="cover" src="https://pic.imgdb.cn/item/669b8708d9c307b7e961166f.png" alt="cover"><div class="content is-center"><div class="date"><i class="far fa-calendar-alt fa-fw"></i> 2024-07-20</div><div class="title">表观基因组学</div></div></a></div><div><a href="/2024/07/20/%5B%E7%94%9F%E4%BF%A1%E7%AC%94%E8%AE%B0%5D%E8%BD%AC%E5%BD%95%E7%BB%84%E5%AD%A6/" title="转录组学"><img class="cover" src="https://pic.imgdb.cn/item/669a4645d9c307b7e9f4ce5f.png" alt="cover"><div class="content is-center"><div class="date"><i class="far fa-calendar-alt fa-fw"></i> 2024-07-20</div><div class="title">转录组学</div></div></a></div><div><a href="/2024/11/21/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5DGenomic%20and%20metabolomic%20insights%20into%20the%20selection%20and%20differentiation%20of%20bioactive%20compounds%20in%20citrus/" title="《Genomic and metabolomic insights into the selection and differentiation of bioactive compounds in citrus》论文精读"><img class="cover" src="/img/cover3.jpg" alt="cover"><div class="content is-center"><div class="date"><i class="far fa-calendar-alt fa-fw"></i> 2024-11-21</div><div class="title">《Genomic and metabolomic insights into the selection and differentiation of bioactive compounds in citrus》论文精读</div></div></a></div></div></div><hr class="custom-hr"/><div id="post-comment"><div class="comment-head"><div class="comment-headline"><i class="fas fa-comments fa-fw"></i><span> 评论</span></div></div><div class="comment-wrap"><div><div id="lv-container" data-id="city" data-uid="MTAyMC82MDAwNC8zNjQ2Nw=="></div></div></div></div></div><div class="aside-content" id="aside-content"><div class="card-widget card-info"><div class="is-center"><div class="avatar-img"><img src="/./img/avatar.jpg" onerror="this.onerror=null;this.src='/img/friend_404.gif'" alt="avatar"/></div><div class="author-info__name">wanzi</div><div class="author-info__description">一个人类</div></div><div class="card-info-data site-data is-center"><a href="/archives/"><div class="headline">文章</div><div class="length-num">41</div></a><a href="/tags/"><div class="headline">标签</div><div class="length-num">35</div></a><a href="/categories/"><div class="headline">分类</div><div class="length-num">10</div></a></div><a id="card-info-btn" target="_blank" rel="noopener" href="https://github.com/wanziw"><i class="fab fa-github"></i><span>Follow Me</span></a><div class="card-info-social-icons is-center"><a class="social-icon" href="https://github.com/wanziw" target="_blank" title="Github"><i class="fab fa-github" style="color: #24292e;"></i></a><a class="social-icon" href="mailto:1530502197@qq.com" target="_blank" title="Email"><i class="fas fa-envelope" style="color: #4a7dbe;"></i></a><a class="social-icon" href="https://blog.csdn.net/weixin_57345774" target="_blank" title="博客"><i class="fas fa-blog"></i></a></div></div><div class="card-widget card-announcement"><div class="item-headline"><i class="fas fa-bullhorn fa-shake"></i><span>公告</span></div><div class="announcement_content">许多东西,我曾经放下,但奇迹般地重新捡了起来</div></div><div class="sticky_layout"><div class="card-widget" id="card-toc"><div class="item-headline"><i class="fas fa-stream"></i><span>目录</span><span class="toc-percentage"></span></div><div class="toc-content"><ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link"><span class="toc-text">代谢组学综述</span></a></li><li class="toc-item toc-level-1"><a class="toc-link"><span class="toc-text">Metabolic GWAS-based dissection of genetic bases underlying the diversity of plant metabolism</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Introduction"><span class="toc-text">Introduction</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#The-Development-of-GWAS"><span class="toc-text">The Development of GWAS</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#The-Rise-of-Metabolic-Diversity"><span class="toc-text">The Rise of Metabolic Diversity</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#Deciphering-the-Genetic-Bases-of-Metabolic-Diversity-with-mGWAS"><span class="toc-text">Deciphering the Genetic Bases of Metabolic Diversity with mGWAS</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link"><span class="toc-text">Limitations and advantages of using metabolite-based genome-wide association studies: Focus on fruit quality traits</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#Introduction-2"><span class="toc-text">Introduction</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#2-%E9%99%90%E5%88%B6mGWAS%E7%9A%84%E5%88%86%E6%9E%90%E5%9B%A0%E7%B4%A0"><span class="toc-text">2 限制mGWAS的分析因素</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#2-5-GWAS-analysis-to-identify-metabolic-candidate-genes-underlying-fruit-quality-traits"><span class="toc-text">2.5 GWAS analysis to identify metabolic candidate genes underlying fruit quality traits</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-6-Metabolomics-based-GWAS-in-citrus-fruits"><span class="toc-text">2.6 Metabolomics-based GWAS in citrus fruits</span></a></li></ol></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link"><span class="toc-text">综述内容</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#%E6%A1%86%E6%9E%B6"><span class="toc-text">框架</span></a></li></ol></li></ol></div></div><div class="card-widget card-recent-post"><div class="item-headline"><i class="fas fa-history"></i><span>最新文章</span></div><div class="aside-list"><div class="aside-list-item"><a class="thumbnail" href="/2025/01/16/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5DSystematic%20identification%20of%20wheat%20spike%20developmental%20regulators%20by%20integrated%20multi-omics,%20transcriptional%20network,%20GWAS,%20and%20genetic%20analyses/" title="《Systematic identification of wheat spike developmental regulators by integrated multi-omics, transcriptional network, GWAS, and genetic analyses》论文阅读"><img src="/img/cover25.jpg" onerror="this.onerror=null;this.src='/img/404.jpg'" alt="《Systematic identification of wheat spike developmental regulators by integrated multi-omics, transcriptional network, GWAS, and genetic analyses》论文阅读"/></a><div class="content"><a class="title" href="/2025/01/16/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5DSystematic%20identification%20of%20wheat%20spike%20developmental%20regulators%20by%20integrated%20multi-omics,%20transcriptional%20network,%20GWAS,%20and%20genetic%20analyses/" title="《Systematic identification of wheat spike developmental regulators by integrated multi-omics, transcriptional network, GWAS, and genetic analyses》论文阅读">《Systematic identification of wheat spike developmental regulators by integrated multi-omics, transcriptional network, GWAS, and genetic analyses》论文阅读</a><time datetime="2025-01-16T09:17:35.000Z" title="发表于 2025-01-16 17:17:35">2025-01-16</time></div></div><div class="aside-list-item"><a class="thumbnail" href="/2025/01/15/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5D%E6%A4%8D%E7%89%A9%E7%9B%B8%E5%85%B3%E5%88%86%E5%AD%90%E6%9C%BA%E5%88%B6%E5%A4%9A%E7%BB%84%E5%AD%A6%E5%88%86%E6%9E%90%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB/" title="植物相关分子机制多组学分析论文阅读"><img src="/img/cover26.jpg" onerror="this.onerror=null;this.src='/img/404.jpg'" alt="植物相关分子机制多组学分析论文阅读"/></a><div class="content"><a class="title" href="/2025/01/15/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5D%E6%A4%8D%E7%89%A9%E7%9B%B8%E5%85%B3%E5%88%86%E5%AD%90%E6%9C%BA%E5%88%B6%E5%A4%9A%E7%BB%84%E5%AD%A6%E5%88%86%E6%9E%90%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB/" title="植物相关分子机制多组学分析论文阅读">植物相关分子机制多组学分析论文阅读</a><time datetime="2025-01-15T02:36:21.000Z" title="发表于 2025-01-15 10:36:21">2025-01-15</time></div></div><div class="aside-list-item"><a class="thumbnail" href="/2025/01/14/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5DSystematic%20classification%20of%20unknown%20metabolites%20using%20high-resolution%20fragmentation%20mass%20spectra/" title="《Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra》论文阅读"><img src="/img/cover24.jpg" onerror="this.onerror=null;this.src='/img/404.jpg'" alt="《Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra》论文阅读"/></a><div class="content"><a class="title" href="/2025/01/14/%5B%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB%5DSystematic%20classification%20of%20unknown%20metabolites%20using%20high-resolution%20fragmentation%20mass%20spectra/" title="《Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra》论文阅读">《Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra》论文阅读</a><time datetime="2025-01-14T09:20:30.000Z" title="发表于 2025-01-14 17:20:30">2025-01-14</time></div></div><div class="aside-list-item"><a class="thumbnail" href="/2025/01/14/%5BDebug%5D%E5%9B%A0%E4%B8%BAVPN%E5%AF%BC%E8%87%B4git-pull-push%E5%A4%B1%E8%B4%A5/" title="因为VPN导致git pull/push失败"><img src="/img/cover3.jpg" onerror="this.onerror=null;this.src='/img/404.jpg'" alt="因为VPN导致git pull/push失败"/></a><div class="content"><a class="title" href="/2025/01/14/%5BDebug%5D%E5%9B%A0%E4%B8%BAVPN%E5%AF%BC%E8%87%B4git-pull-push%E5%A4%B1%E8%B4%A5/" title="因为VPN导致git pull/push失败">因为VPN导致git pull/push失败</a><time datetime="2025-01-14T04:28:44.000Z" title="发表于 2025-01-14 12:28:44">2025-01-14</time></div></div><div class="aside-list-item"><a class="thumbnail" href="/2025/01/13/%5BWeb%E5%BC%80%E5%8F%91%5Djquery%E5%AD%A6%E4%B9%A0/" title="JQuery学习"><img src="/img/cover22.jpg" onerror="this.onerror=null;this.src='/img/404.jpg'" alt="JQuery学习"/></a><div class="content"><a class="title" href="/2025/01/13/%5BWeb%E5%BC%80%E5%8F%91%5Djquery%E5%AD%A6%E4%B9%A0/" title="JQuery学习">JQuery学习</a><time datetime="2025-01-13T03:52:20.000Z" title="发表于 2025-01-13 11:52:20">2025-01-13</time></div></div></div></div></div></div></main><footer id="footer"><div id="footer-wrap"><div class="copyright">&copy;2024 - 2025 By wanzi</div><div class="framework-info"><span>框架 </span><a target="_blank" rel="noopener" href="https://hexo.io">Hexo</a><span class="footer-separator">|</span><span>主题 </span><a target="_blank" rel="noopener" href="https://github.com/jerryc127/hexo-theme-butterfly">Butterfly</a></div><div class="footer_custom_text">很高心认识你！</div></div></footer></div><div id="rightside"><div id="rightside-config-hide"><button id="readmode" type="button" title="阅读模式"><i class="fas fa-book-open"></i></button><button id="darkmode" type="button" title="浅色和深色模式转换"><i class="fas fa-adjust"></i></button><button id="hide-aside-btn" type="button" title="单栏和双栏切换"><i class="fas fa-arrows-alt-h"></i></button></div><div id="rightside-config-show"><button id="rightside-config" type="button" title="设置"><i class="fas fa-cog fa-spin"></i></button><button class="close" id="mobile-toc-button" type="button" title="目录"><i class="fas fa-list-ul"></i></button><a id="to_comment" href="#post-comment" title="直达评论"><i class="fas fa-comments"></i></a><button id="go-up" type="button" title="回到顶部"><span class="scroll-percent"></span><i class="fas fa-arrow-up"></i></button></div></div><div><script src="/js/utils.js?v=4.13.0"></script><script src="/js/main.js?v=4.13.0"></script><script src="https://cdn.jsdelivr.net/npm/@fancyapps/ui@5.0.33/dist/fancybox/fancybox.umd.min.js"></script><script src="https://cdn.jsdelivr.net/npm/instant.page@5.2.0/instantpage.min.js" type="module"></script><script>function panguFn () {
  if (typeof pangu === 'object') pangu.autoSpacingPage()
  else {
    getScript('https://cdn.jsdelivr.net/npm/pangu@4.0.7/dist/browser/pangu.min.js')
      .then(() => {
        pangu.autoSpacingPage()
      })
  }
}

function panguInit () {
  if (true){
    GLOBAL_CONFIG_SITE.isPost && panguFn()
  } else {
    panguFn()
  }
}

document.addEventListener('DOMContentLoaded', panguInit)</script><div class="js-pjax"><script>(()=>{
  const loadLivere = () => {
    if (typeof LivereTower === 'object') window.LivereTower.init()
    else {
      (function(d, s) {
          var j, e = d.getElementsByTagName(s)[0];
          if (typeof LivereTower === 'function') { return; }
          j = d.createElement(s);
          j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
          j.async = true;
          e.parentNode.insertBefore(j, e);
      })(document, 'script');
    }
  }

  if ('Livere' === 'Livere' || !false) {
    if (false) btf.loadComment(document.getElementById('lv-container'), loadLivere)
    else loadLivere()
  } else {
    window.loadOtherComment = loadLivere
  }
})()</script></div><script defer="defer" id="ribbon" src="https://cdn.jsdelivr.net/npm/butterfly-extsrc@1.1.3/dist/canvas-ribbon.min.js" size="150" alpha="0.6" zIndex="-1" mobile="true" data-click="true"></script><script id="canvas_nest" defer="defer" color="0,0,255" opacity="0.7" zIndex="-1" count="99" mobile="false" src="https://cdn.jsdelivr.net/npm/butterfly-extsrc@1.1.3/dist/canvas-nest.min.js"></script><script src="https://cdn.jsdelivr.net/npm/pjax@0.2.8/pjax.min.js"></script><script>let pjaxSelectors = ["meta[property=\"og:image\"]","meta[property=\"og:title\"]","meta[property=\"og:url\"]","head > title","#config-diff","#body-wrap","#rightside-config-hide","#rightside-config-show",".js-pjax"]

var pjax = new Pjax({
  elements: 'a:not([target="_blank"])',
  selectors: pjaxSelectors,
  cacheBust: false,
  analytics: false,
  scrollRestoration: false
})

document.addEventListener('pjax:send', function () {

  // removeEventListener
  btf.removeGlobalFnEvent('pjax')
  btf.removeGlobalFnEvent('themeChange')

  document.getElementById('rightside').classList.remove('rightside-show')
  
  if (window.aplayers) {
    for (let i = 0; i < window.aplayers.length; i++) {
      if (!window.aplayers[i].options.fixed) {
        window.aplayers[i].destroy()
      }
    }
  }

  typeof typed === 'object' && typed.destroy()

  //reset readmode
  const $bodyClassList = document.body.classList
  $bodyClassList.contains('read-mode') && $bodyClassList.remove('read-mode')

  typeof disqusjs === 'object' && disqusjs.destroy()
})

document.addEventListener('pjax:complete', function () {
  window.refreshFn()

  document.querySelectorAll('script[data-pjax]').forEach(item => {
    const newScript = document.createElement('script')
    const content = item.text || item.textContent || item.innerHTML || ""
    Array.from(item.attributes).forEach(attr => newScript.setAttribute(attr.name, attr.value))
    newScript.appendChild(document.createTextNode(content))
    item.parentNode.replaceChild(newScript, item)
  })

  GLOBAL_CONFIG.islazyload && window.lazyLoadInstance.update()

  typeof panguInit === 'function' && panguInit()

  // google analytics
  typeof gtag === 'function' && gtag('config', '', {'page_path': window.location.pathname});

  // baidu analytics
  typeof _hmt === 'object' && _hmt.push(['_trackPageview',window.location.pathname]);

  typeof loadMeting === 'function' && document.getElementsByClassName('aplayer').length && loadMeting()

  // prismjs
  typeof Prism === 'object' && Prism.highlightAll()
})

document.addEventListener('pjax:error', e => {
  if (e.request.status === 404) {
    pjax.loadUrl('/404.html')
  }
})</script><script async data-pjax src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script><div id="local-search"><div class="search-dialog"><nav class="search-nav"><span class="search-dialog-title">搜索</span><span id="loading-status"></span><button class="search-close-button"><i class="fas fa-times"></i></button></nav><div class="is-center" id="loading-database"><i class="fas fa-spinner fa-pulse"></i><span>  数据库加载中</span></div><div class="search-wrap"><div id="local-search-input"><div class="local-search-box"><input class="local-search-box--input" placeholder="搜索文章" type="text"/></div></div><hr/><div id="local-search-results"></div><div id="local-search-stats-wrap"></div></div></div><div id="search-mask"></div><script src="/js/search/local-search.js?v=4.13.0"></script></div></div></body></html>